pkg/config/engine/containerd/config_v1.go

```diff
@@ -22,13 +22,29 @@ import (
 	"github.com/pelletier/go-toml"
 
 	"github.com/NVIDIA/nvidia-container-toolkit/pkg/config/engine"
+	cfgtoml "github.com/NVIDIA/nvidia-container-toolkit/pkg/config/toml"
```

Comment:
I think I missed simply removing the:
```
	"github.com/pelletier/go-toml"
```
import here and replacing it with:
```suggestion
	"github.com/NVIDIA/nvidia-container-toolkit/pkg/config/toml"
```

Let's apply the following diff:
```
diff --git a/pkg/config/engine/containerd/config_v1.go b/pkg/config/engine/containerd/config_v1.go
index 564a2ae9..8d756ea2 100644
--- a/pkg/config/engine/containerd/config_v1.go
+++ b/pkg/config/engine/containerd/config_v1.go
@@ -19,10 +19,8 @@ package containerd
 import (
 	"fmt"
 
-	"github.com/pelletier/go-toml"
-
 	"github.com/NVIDIA/nvidia-container-toolkit/pkg/config/engine"
-	cfgtoml "github.com/NVIDIA/nvidia-container-toolkit/pkg/config/toml"
+	"github.com/NVIDIA/nvidia-container-toolkit/pkg/config/toml"
 )
 
 // ConfigV1 represents a version 1 containerd config
@@ -31,7 +29,7 @@ type ConfigV1 Config
 var _ engine.Interface = (*ConfigV1)(nil)
 
 type ctrdCfgV1Runtime struct {
-	tree *cfgtoml.Tree
+	tree *toml.Tree
 }
 
 var _ engine.RuntimeConfig = (*ctrdCfgV1Runtime)(nil)
```